test/features/schema_tests.js

```diff
@@ -33,7 +38,7 @@ describe('Responses should conform to the provided JSON schema of the responce',
     });
 
     it('/feed/featured should conform schema, ruwiki', function() {
-        return preq.get({ uri: server.config.hostPort + '/ru.wikipedia.org/v1/feed/featured/2016/09/08' })
+        return preq.get({ uri: `${server.config.hostPort}/ru.wikipedia.org/v1/feed/featured/${getToday()}` })
```

Comment:
Minor: Could use an array for the uri instead of a string. This saves the need to parse from the string.